We do have inttypes.h, but gcc-4.2.1 doesn't default to C99 and
thus doesn't #define __STDC_VERSION__ 199901L.

Index: src/duk_config.h
--- src/duk_config.h.orig
+++ src/duk_config.h
@@ -1644,6 +1644,10 @@
 #define DUK_F_HAVE_INTTYPES
 #endif
 
+#ifndef DUK_F_HAVE_INTTYPES
+#define DUK_F_HAVE_INTTYPES
+#endif
+
 /* Basic integer typedefs and limits, preferably from inttypes.h, otherwise
  * through automatic detection.
  */
